performance/fhir-meter/README.md

```diff
@@ -25,3 +25,160 @@ Also the following variables will need to be filled in or verified:
 You can toggle any of the tests enabled or disabled as desired to run the load you would like on the server.
 
 The JMeter listeners added are standard JMeter listeners.  Feel free to add your own listener to suit your needs.
+
+Here is a list of searches that make up the JMeter test plan. All data references are mock data.
+
+[base]/ExplanationOfBenefit?patient=Patient/id-abc-123&created=ge2001&created=lt2023&_sort=-created&_count=10
```

Comment:
i suggest wrapping this in a code block (3 backticks).  then each row in the UI should be a single query.

alternatively, you could make the whole thing a list.  your call.